apps/condo/domains/marketplace/components/MarketplaceOnboarding/RequisitesSetup.tsx

```diff
@@ -139,7 +147,7 @@ export const RequisitesSetup: React.FC = () => {
                 bic: get(acquiringContext, ['invoiceRecipient', 'bic'], ''),
                 account: get(acquiringContext, ['invoiceRecipient', 'bankAccount'], ''),
                 taxType: get(acquiringContext, 'invoiceTaxRegime'),
-                taxPercent: get(acquiringContext, 'invoiceVatPercent'),
+                taxPercent: Number(get(acquiringContext, 'invoiceVatPercent', '')).toString(),
```

Comment:
It seems that a type error will occur when creating “without VAT”. Empty string invalid value for `Decimal` type
<img width="1679" alt="image" src="https://github.com/open-condo-software/condo/assets/64303474/a6a1676d-ac0f-435f-99e1-9f596e5d09b6">

Reply:
This casting type is needed to convert '10.0000' to '10' to set the previously selected value within the option group. The database stores the `Decimal` type converted to string at the front-end ('10.0000'). We have options for VAT as numbers without zeros at the end ('10').

Reply:
Fixed. Now pass `null` instead of `''`